src/iexamine.cpp

```diff
@@ -1222,6 +1222,10 @@ void iexamine::vending( Character &you, const tripoint_bub_ms &examp )
     int money = you.charges_of( itype_cash_card );
     map_stack vend_items = get_map().i_at( examp );
 
+    if( get_option<bool>( "CAPITALISM" ) ) {
+        money = you.cash;
```

Comment:
```suggestion
        money = you.cash + you.charges_of( itype_cash_card );
```

Otherwise cash cards will never be recognized. If you walked in with with $1,000 on cash cards and $0 in the bank it'd always pretend you had $0.

I assume cash cards are a thing in Aftershock. Please ignore this suggestion if not.